mcs/class/Mono.Debugger.Soft/Test/dtest.cs

```diff
@@ -5044,6 +5044,48 @@ public void Bug59649 ()
 
 		ShouldNotSendUnexpectedTypeLoadEventsAndInvalidSuspendPolicyAfterAttach (port);
 	}
+
+	[Test]
+	public void InvokeMethodFixedArray () {
+		Event e = run_until ("fixed_size_array");
+		var req = create_step (e);
+		req.Enable ();
+		step_once ();
+		step_over ();
+		step_over ();
+		var bp = step_over ();
+		var thread = bp.Thread;
+		var frame = thread.GetFrames()[0];
+
+		var local =  frame.GetVisibleVariableByName("n");
```

Comment:
Where does this test fixed size arrays ?

Reply:
Here:
public unsafe struct NodeTestFixedArray
{
	private fixed short buffer[4];
	private fixed char buffer2[4];